bundles/org.openhab.binding.solarman/src/main/resources/definitions/deye_sg04lp3.yaml

```diff
@@ -3,13 +3,13 @@
 
 requests:
   - start: 0x0003
-    end: 0x0059
+    end: 0x0011
     mb_functioncode: 0x03
   - start: 0x0063
     end: 0x006D
     mb_functioncode: 0x03
   - start: 0x0085
-    end: 0x0085
+    end: 0x0086
```

Comment:
Same question.

Reply:
It did not work with 0x0085, the actual implementation is not able to handel just 1 byte. I will check this.

Reply:
I made it work with only one register